packages/mobile/src/escrow/saga.ts

```diff
@@ -321,11 +539,21 @@ export function* watchFetchSentPayments() {
 export function* watchVerificationEnd() {
   while (true) {
     const update: SetVerificationStatusAction = yield take(IdentityActions.SET_VERIFICATION_STATUS)
+    const feelessUpdate: FeelessSetVerificationStatusAction = yield take(
+      IdentityActions.FEELESS_SET_VERIFICATION_STATUS
+    )
```

Comment:
You probably want a `race` between these two. Otherwise it will wait for _both_ and I imagine only one or the other will happen.

Reply:
Good point